libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.m

```diff
@@ -371,6 +434,32 @@ - (void)logoutUser:(SFUserAccount *)user {
     }];
 }
 
+- (void)revokeRefreshToken:(SFOAuthCredentials *)credentials {
```

Comment:
This should probably move to the `OAuthClient` class - it's related to credentials, not really associated with the concept of a logged in user.